tensorboard/webapp/metrics/views/card_renderer/scalar_card_component.ts

```diff
@@ -97,6 +100,12 @@ export class ScalarCardComponent<Downloader> {
 
   yScaleType = ScaleType.LINEAR;
   isViewBoxOverridden: boolean = false;
+  dataHeaders: ColumnHeaders[] = [
+    ColumnHeaders.RUN,
+    ColumnHeaders.VALUE,
+    ColumnHeaders.STEP,
+    ColumnHeaders.RELATIVE_TIME,
+  ];
```

Comment:
Just a thought, do you expect this array to be dynamic changing or a constant? 

Not required in this PR but I might prefer is as as a constant and put this outside of the scalar_card_component. Later in range selection, we put more attributes in the array but the data_table_component is smart enough to not render the column if there is no such key provided in data (type SelectedStepRunData). That is to say, in single selection  we have 'VALUE_CHANGE' in 'headers' but not in keys of SelectedStepRunData type object so the data_component will not render this column.